src/summarycode/todo.py

```diff
@@ -213,6 +213,8 @@ def get_package_identifier(package_data, file_path):
 
 
 def get_unknown_purl(package_type):
+    if not package_type:
```

Comment:
Can you elaborate about this unknown purl?

Reply:
Removed this, was being required in case of JARs where we did not have a `default_package_type`